### PR TITLE
find: Add param 'min_depth' to limit search

### PR DESCRIPTION
**Pull request by Marschelloss**
##### SUMMARY
The cli-tool `find` has the helpful parameters `-mindepth` and `-maxdepth` to limit the search scope. It would be great to use these paramaters also in the ansible module `find`. So I added the possible parameter `min_depth`.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
`find.py`

##### ADDITIONAL INFORMATION
Since I didn't want to break other playbooks, I didn't rename the parameter `depth` to `max_depth`, and only added the `min_depth` parameter. But I am unsure how the ansible community would handle these things (maybe switch to `max` and `min`, but still parse `depth` as a fallback).

### CODE REVIEW

File: lib/ansible/modules/files/find.py [hunk 1]
```diff
@@ -372,7 +377,8 @@ def main():
             follow=dict(type='bool', default=False),
             get_checksum=dict(type='bool', default=False),
             use_regex=dict(type='bool', default=False),
-            depth=dict(type='int'),
```
Comment (sivel) [inline lib/ansible/modules/files/find.py:375]: You cannot remove existing params, they either need to be an alias, or it needs to toggle different functionality.

**Review by sivel — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/files/find.py (hunk 1 above)
Comment (Marschelloss) [reply lib/ansible/modules/files/find.py:375]: Of course you are right, old playbooks shouldn't be affected of this change. Like i mentioned in my initial post, I was unsure if I should add the new params 'max' and 'min' or if I should only add 'min' and let 'depth' unchanged. 
Seems like while trying out both options I missed something in my code. I pushed a new commit to fix this, and only add 'min_depth'. 

Or would be an alias the better option?

Comment (bcoca) [reply lib/ansible/modules/files/find.py:375]: an alias would work

**Comment by bcoca** [comment]
waiting_on_contributor

**Pull request closed without merging**